specification/informatica/Informatica.DataManagement.Management/tspconfig.yaml

```diff
@@ -1,6 +1,6 @@
 parameters:
   "service-dir":
-    default: "sdk/informatica"
+    default: "sdk/informaticadatamanagement"
```

Comment:
why this is different?

Reply:
if this is only for python and java, i prefer the original default which makes sense to me.

Reply:
The failure is caused by out automation to handle the different service-dir and here is the issue to track that: https://github.com/Azure/azure-sdk-tools/issues/9946.

Reply:
> if this is only for python and java, i prefer the original default which makes more sense to me.

Original default is `informaticadatamanagement`, as approved by namespace review:
https://github.com/Azure/azure-sdk-pr/issues/1731

`service-dir` usually conforms to package name, unless specifically mentioned otherwise, and I haven't seen such case before..

Reply:
.net 
https://github.com/Azure/azure-sdk-for-net/tree/main/sdk/informaticadatamanagement